Test/Common/DomainTest/implementation/VolunteerOpportunityTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Ttu.Domain;
+
+namespace Ttu.DomainTest.implementation
+{
+    [TestClass]
+    public class VolunteerOpportunityTest
+    {
+        private VolunteerOpportunity _volunteerOpportunity;
```

Comment:
The underscore prefix is kind of an older convention. You might look around the DomainTest project a little bit and you'll see some of the newer conventions.

I like to have all instance variables start with an uppercase variable name and all lowercase variables start with a lowercase variable name. This helps be know what is a variable for the object as a whole and what is just part of a limited scope within the class.